computer-vision.cfg

```diff
@@ -1,2362 +0,0 @@
-# Doxyfile 1.8.9.1
```

Comment:
Doxygen doesn't modify header files, it reads the header files and creates nicely formatted documentation with the help of doxygen-style comments in the code. You have to write the descriptions yourself.
For example, this is the generated documentation for the project: https://uwarg.github.io/computer-vision/html/index.html